config/locales/en.yml

```diff
@@ -214,8 +214,12 @@ en:
         under_hard_min: "Net income cannot be less than £%{hard_min} per week given the tenant’s working situation"
         freq_missing: "Select how often the household receives income"
         earnings_missing: "Enter how much income the household has in total"
-      income1:
-        over_hard_max: "Income must be lower than £%{hard_max}"
+      income:
+        over_hard_max_for_london: "Income must not exceed £90,000 for properties within London local authorities"
+        over_hard_max_for_outside_london: "Income must not exceed £80,000 for properties outside London local authorities"
+        combined_over_hard_max_for_london: "Combined income must not exceed £90,000 for properties within London local authorities"
```

Comment:
nit: the copy I'm looking at uses "The combined income..." rather than "Combined income..."